js/main.js

```diff
@@ -0,0 +1,75 @@
+/*
+Aziz Omar
+Chatbot2
+*/
+// contains all inputs and outputs
+let data = [{
+        input: 'Hi',
+        output: ['Hello', 'Hey !', 'Whats up?']
+    },
+
+    {
+        input: 'How are you?',
+        output: ['good', 'not bad', 'doing well']
+    },
+
+    {
+        input: 'What is your favourite color?',
+        output: ['red', 'gray', 'black']
+    },
+
+    {
+        input: 'Where are you',
+        output: ['home', 'at work', 'on  vacation']
+    }
+];
+
+
+/*
+iterates through an array to find a question simillar to user's input
+*/
+
+const findSimillarQuestion = givenInput => (item) => {
+ //   return item.input.toUpperCase().includes(givenInput.toUpperCase());
+    return item.input.toUpperCase() == givenInput.toUpperCase();
+}
+
+
+
+// repaly function
+const replay = () => {
+    let question = document.getElementById("input").value;
+    let txtArea = document.getElementById("output");
+    let rndmNumber = Math.floor(Math.random() * data[0].output.length);
+    let detectedQuestion = data.filter(findSimillarQuestion(question))[0];
+
+    if (detectedQuestion) {
+         let maxAnswer=detectedQuestion.output.sort()[detectedQuestion.output.length-1];
+         let minAnswer=detectedQuestion.output.sort()[0]
+
+        //assign the longest answer to maxAnswer
```

Comment:
When you're developing, it helps to comment out code to test things. But when you submit your code for review, it's a good idea to remove the commented out code to keep things neat and tidy.